apps/src/blocklyAddons/cdoTrashcan.js

```diff
@@ -0,0 +1,103 @@
+import GoogleBlockly from 'blockly/core';
+
+export default class CdoTrashcan extends GoogleBlockly.Trashcan {
+  /** Use our trash png and add ⃠ for undeletable blocks
+   * @override
+   */
+  createDom() {
+    super.createDom();
+
+    // Use our trashcan png instead of Google's
+    this.svgGroup_.childNodes.forEach(node => {
+      if (node.nodeName === 'image') {
+        node.setAttributeNS(
+          Blockly.utils.dom.XLINK_NS,
+          'xlink:href',
+          CdoTrashcan.TRASH_URL
+        );
+      }
+    });
+
+    this.notAllowed_ = Blockly.utils.dom.createSvgElement(
+      'g',
+      {},
+      this.svgGroup_
+    );
+    Blockly.utils.dom.createSvgElement(
+      'line',
+      {x1: 0, y1: 10, x2: 45, y2: 60, stroke: '#c00', 'stroke-width': 5},
+      this.notAllowed_
+    );
+    Blockly.utils.dom.createSvgElement(
+      'circle',
+      {cx: 22, cy: 33, r: 33, stroke: '#c00', 'stroke-width': 5, fill: 'none'},
+      this.notAllowed_
+    );
```

Comment:
Are these the circle with the line going through it?

Reply:
Yep, looks like this:
![Oct-05-2020 13-00-26](https://user-images.githubusercontent.com/8787187/95126190-e94ae880-070a-11eb-86f7-0fb65001af45.gif)